config.yaml.sample

```diff
@@ -206,6 +209,10 @@ toi-prune:
     path: osm
     layer: all
     format: zip
+  # a reduced version of prev s3 entity
```

Comment:
Can you make this comment more generic (so it's less about what _was_ and more about what the following config is for)?